pse/p0tionConfig.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Ethstorage Circuits Ceremony",
+  "description": "This is a trusted setup ceremony for the Ethstorage Circuits with is used to proof storage on L2 dynamic datasets with an Ethereum L1 Contract",
```

Comment:
This is a trusted setup ceremony for the Ethstorage Circuits which is used to prove storage on L2 dynamic datasets with an Ethereum L1 Contract